### PR TITLE
Update User Guide - Street Smart API support

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Add Cyclomedia tool on [Street View](https://mapstore--10021.org.readthedocs.build/en/10021/user-guide/street-view/#cyclomedia) section

### CODE REVIEW

**Review by tdipisa — CHANGES_REQUESTED** [review]

File: docs/user-guide/street-view.md [hunk 1]
```diff
@@ -22,5 +29,43 @@ By clicking on a street in the map, the tool window displays the Street View and
...
+
+<video class="ms-docimage" controls><source src="../img/street-view/add-point.mp4"/></video>
+
+By clicking on a highlighted point on the map, the tool window displays the Street View and the user can navigate it as usual or interact with the *Navigation Toolbar*.
+
+<img src="../img/street-view/cyclomedia_navigation_bar.jpg" class="ms-docimage" width="500px"/>
+
+Through the navigation toolbar it is possible to:
+
+* Open the **Overlays** pop-up, through the <img src="../img/button/overlays-button.jpg" class="ms-docbutton"/> button, to: view and enable/disable the layers present in the view in the **Layers** tab, enable/disable the *Show Mouse position* and the *Show Compass* in the **Viewer** tab.
```
[the remark this answers is not in the document]
Comment (ElenaGallo) [reply docs/user-guide/street-view.md:51]: * Open the **Overlays** pop-up, through the  button, to:

* View and enable/disable the layers present in the view in the **Layers** tab

* Enable/disable the *Show Mouse position* and the *Show Compass* in the **Viewer** tab

File: docs/user-guide/street-view.md [hunk 2]
```diff
@@ -22,5 +29,54 @@ By clicking on a street in the map, the tool window displays the Street View and
...
 
+## Cyclomedia Street Smart
+
+If the [Cyclomedia Street Smart](https://www.cyclomedia.com/en/street-smart) has been configured as a provider for the MS Street View tool, the user experience is the same: the tool can be activated through the same button <img src="../img/button/street-view-button.jpg" class="ms-docbutton"/> in [Side Toolbar](mapstore-toolbars.md#side-toolbar).
+
+<img src="../img/street-view/cyclomedia-popup.jpg" class="ms-docimage" width="500px"/>
+
 !!! note
-    By default, the **Street View** plugin is ready to be configured for [application contexts](application-context.md#configure-plugins), and is not available in the default plugin configuration due to licensing reasons.
+    The **[Cyclomedia Street Smart](https://www.cyclomedia.com/en/street-smart)** provided can be configured on the *Street View* plugin as documented [here](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.StreetView)
```
Comment (tdipisa) [inline docs/user-guide/street-view.md:39]:
```suggestion
    The **[Cyclomedia Street Smart](https://www.cyclomedia.com/en/street-smart)** provider can be configured on the *Street View* plugin as documented [here](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.StreetView)
```

File: docs/user-guide/street-view.md [hunk 3]
```diff
@@ -22,5 +29,54 @@ By clicking on a street in the map, the tool window displays the Street View and
...
+
+When the tool is activated, the user must enter *Username* and *Password* and click <img src="../img/button/submit-button.jpg" class="ms-docbutton"/>.
+
+<img src="../img/street-view/credentials_api.jpg" class="ms-docimage" width="500px"/>
+
+!!! warning
+    The credentials are necessary to visualize the street layer while the usual authkey is needed in plugin config to interact with the street smart APIs (see plugin config page [here](https://dev-mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.StreetView))
```
Comment (tdipisa) [inline docs/user-guide/street-view.md:46]:
```suggestion
    The credentials are necessary to visualize the street layer while the usual authkey is needed in plugin config to interact with the street smart APIs (see plugin config page [here](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.StreetView))
```

**Review by tdipisa — CHANGES_REQUESTED** [review]

**Review by tdipisa — APPROVED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo please backport to 2024.01.xx. Thank you

**Pull request merged**